preview-docs/Webhooks.md

```diff
@@ -0,0 +1,79 @@
+# RFC: Webhooks for schema change notifications
+
+To help you better integrate Apollo Studio into your unique workflow, we are starting some work to offer standard webhooks for certain events. The first of those events will be schema changes - any time we detect a change to the schema for your graph.
```

Comment:
```suggestion
To help you better integrate Apollo Studio into your unique workflow, we are starting some work to offer standard webhooks for certain events. The first of those events will be schema changes - any time a new schema is published to your graph.
```

Reply:
Do we send an event if you push a schema with no changes?